### PR TITLE
stack along channels in opStackLoader

### PR DESCRIPTION
**Pull request by FynnBe**
Relates to  ilastik/ilastik#1202

### CODE REVIEW

**Comment by stuarteberg** [comment]
Thanks! Can you add a test case to `testOpStackLoader.py` to exercise the new functionality?

PS -- I think you should avoid writing "Fixes..." in the PR description here, for two reasons:

1. That issue number refers to a different repo (ilastik, not lazyflow), so it is not clear what issue you're talking about.  If you really want to cross-reference that issue from this thread, github supports this syntax: `org/repo#issue`, e.g. ilastik/ilastik#1202.

2. Github will automatically close the issue if it detects the words "Fixes #X" in your PR description.  But in this case, that issue can't be closed until multiple PRs have been merged (this one and also ilastik/ilastik#1526).  So it's probably best to omit that text here and just manually close the issue when we're ready.

**Comment by chaubold** [comment]
I tested it for a few scenarios and it works nicely!

File: lazyflow/operators/ioOperators/ioOperators.py [hunk 1]
```diff
@@ -92,36 +93,42 @@ def setupOutputs(self):
             logger.error(str(e))
             raise OpStackLoader.FileOpenError(self.fileNameList[0])
 
-
         slice_shape = self.info.getShape()
         X, Y, C = slice_shape
```
Comment (FynnBe) [inline lazyflow/operators/ioOperators/ioOperators.py]: here the axis are being interpreted as xy[c]

File: lazyflow/operators/ioOperators/ioOperators.py [hunk 2]
```diff
@@ -92,36 +93,42 @@ def setupOutputs(self):
...
-            shape = (T, Z, Y, X, C)
-            axistags = vigra.defaultAxistags(stack_order)
-            
+                axistags = vigra.defaultAxistags('ztyxc')
+            elif sequence_axis == 't':
+                axistags = vigra.defaultAxistags('tzyxc')
+            else:
+                axistags = vigra.defaultAxistags('czyx')
+
+            # For stacks of 3D volumes, we assume xyz blocks stacked along
```
[the remark this answers is not in the document]
Comment (FynnBe) [reply lazyflow/operators/ioOperators/ioOperators.py]: No, that is correct. See line 97: 
```        X, Y, C = slice_shape```
We then reorder it to t/z-y-x-c.
It is still a bit unclear. This should be revisited, when simplifying overall axis order handling.

**Pull request merged**